addons/events/fnc_addPlayerEventHandler.sqf

```diff
@@ -192,6 +200,12 @@ if (_id != -1) then {
                 GVAR(oldCameraView) = _data;
                 [QGVAR(cameraViewEvent), [_player, _data]] call CBA_fnc_localEvent;
             };
+
+            _data = call CBA_fnc_getActiveFeatureCamera;
```

Comment:
performance?
We might want to only add this when it's actually needed.

Reply:
It's just a bunch of null checks (7 in vanilla). Aren't all Player EHs only added when they are needed already?

Reply:
0.0301 - 0.31 ms on 10k/10k cycles.

Reply:
0.03 ouhhhh.
How about a check whether either the array of eventhandlers is not empty (by using the CBA internal variable name of the array)
Or check that `GVAR(oldFeatureCamera` is not nil? and set it to non nil once a Eventhandler is added in line 92?

Reply:
That wouldn't work for cases where we might want to use internal variable (ref ACE PR).